### PR TITLE
sinogeom :moj plot

### PR DESCRIPTION
**Pull request by JeffFessler**
and code formatting...

### CODE REVIEW

File: src/io/fld-read.jl [hunk 1]
```diff
@@ -247,65 +247,29 @@ end
...
-		format = Float64
-		endian = "ieee-be"
-		bytes = 8
-
-	else
-		throw("format '$datatype' not yet implemented. ask jeff!")
-	end
 
-	return format, endian, bytes
+	dict = Dict([
```
Comment (johnnychen94) [inline src/io/fld-read.jl:254]: Compared to the current implementation, Using `Dict` might be bad in performance.

```julia
# after
julia> @btime datatype_fld_to_mat("byte")
  1.822 μs (60 allocations: 3.81 KiB)
(UInt8, :be, 1)

# Before
julia> @btime datatype_fld_to_mat("byte")
  50.033 ns (1 allocation: 32 bytes)
(UInt8, "ieee-be", 1)
```

Comment (JeffFessler) [reply src/io/fld-read.jl:254]: Thanks very much for the info!  Then I will avoid using `Dict` in performance critical places.  Here it is just some IO for an obscure file type used rarely so I think this version is easier to read.
I'm mostly working to improve code coverage at the moment...

Comment (JeffFessler) [reply src/io/fld-read.jl:254]: BTW, is there a `const` variant that would be better to use?  I couldn't find one...

Comment (johnnychen94) [reply src/io/fld-read.jl:254]: > BTW, is there a const variant that would be better to use? I couldn't find one...

I don't know much about it, but I guess `NamedTuple` might work

Comment (JeffFessler) [reply src/io/fld-read.jl:254]: Thanks - good idea.  Unfortunately the keys have `-` in the string so it would be some work.  Maybe later...

**Comment by JeffFessler** [comment]
I got accidentally got two branches mixed up but in the end this merge should just be sino_geom related.

**Comment by johnnychen94** [comment]
Yes, this is because you created this branch on the top of `fldtest` branch (#63) instead of `master` branch.

FYI, before #63 is merged, you can use `git rebase master` in `sinogeom` branch to not include those new commits in `fldtest`.

**Pull request merged**